termwiz/src/tmux_cc/mod.rs

```diff
@@ -793,6 +795,49 @@ pub fn parse_layout(layout: &str) -> anyhow::Result<Vec<WindowLayout>> {
     Ok(result)
 }
 
+// https://github.com/tmux/tmux/wiki/Control-mode
+// The tmux encode all characters < 32 as Oct '\xxx'
+// This function decode those encoded characters to raw bytes
+fn decode_tmux_output(bytes: &[u8]) -> Vec<u8> {
```

Comment:
how does this differ from the `unvis` function that we have already? `unvis` returns a `String`, so that's an obvious difference, but I'm wondering if this is logically the same function that should be applied in the same places?